runelite-client/src/main/java/net/runelite/client/plugins/music/MusicPlugin.java

```diff
@@ -466,17 +466,20 @@ private void updateList(Widget musicList, String filter)
 
 		public void update()
 		{
-			handle.setOnDragListener((JavaScriptCallback) this::drag);
-			handle.setOnDragCompleteListener((JavaScriptCallback) this::drag);
-			handle.setHasListener(true);
-
-			track.setOnMouseRepeatListener((JavaScriptCallback) ev ->
+			JavaScriptCallback setSliderTooltip = ev ->
 			{
 				int value = channel.getValue();
 				int percent = (int) Math.round((value * 100.0 / channel.getMax()));
 
 				sliderTooltip = new Tooltip(channel.getName() + ": " + percent + "%");
-			});
+			};
+
+			handle.setOnMouseRepeatListener(setSliderTooltip);
```

Comment:
This additionally needs to be unset in shutdown.

Reply:
Yeah I was curious about this. This line actually exists currently (I just refactored it) and was never unset in shutdown. Should we still add that in?

Reply:
This line actually doesn't exist currently

Reply:
Ah right my bad, it only existed on the slider and not the handler.